x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/topn/TopNOperatorTests.java

```diff
@@ -505,7 +520,7 @@ public void testCollectAllValues() {
         encoders.add(DEFAULT_SORTABLE);
 
         for (ElementType e : ElementType.values()) {
-            if (e == ElementType.UNKNOWN || e == COMPOSITE || e == FLOAT) {
+            if (e == ElementType.UNKNOWN || e == COMPOSITE) {
```

Comment:
I added this exclusions for FLOAT when adding FloatBlocks, the exclusions can now be removed.